packages/vite/project.json

```diff
@@ -13,7 +13,7 @@
       "outputs": ["{workspaceRoot}/coverage/packages/vite"]
     },
     "build-base": {
-      "executor": "@nx/js:swc",
+      "executor": "@nx/js:tsc",
```

Comment:
One project needs to use swc so that we can test it out thoroughly by dogfooding it. Is there another project using it?